content/post/2018-04-02-sct-encoding.markdown

```diff
@@ -0,0 +1,439 @@
+
+In the decoded data, we can find the OID 1.3.6.1.4.1.11129.2.4.2, indicating
```

Comment:
I think it would make sense to put OID's in backticks, e.g. `1.3.6.1.4.1.11129.2.4.2`